sdk/keyvault/azcertificates/models.go

```diff
@@ -101,8 +101,35 @@ func certificateAttributesFromGenerated(g *generated.CertificateAttributes) *Cer
 	}
 }
 
-// Certificate - A certificate bundle consists of a certificate (X509) plus its properties.
-type Certificate struct {
+// KeyVaultCertificate - A certificate bundle consists of a certificate (X509) plus its properties.
+type KeyVaultCertificate struct {
+	// The certificate attributes.
+	Properties *CertificateProperties `json:"attributes,omitempty"`
+
+	// CER contents of x509 certificate.
+	Cer []byte `json:"cer,omitempty"`
+
+	// The content type of the secret. eg. 'application/x-pem-file' or 'application/x-pkcs12',
+	ContentType *string `json:"contentType,omitempty"`
+
+	// Application specific metadata in the form of key-value pairs
+	Tags map[string]string `json:"tags,omitempty"`
+
+	// READ-ONLY; The certificate id.
+	ID *string `json:"id,omitempty" azure:"ro"`
```

Comment:
`azure:"ro"`

Whoa, that's neat! Where can I learn more about how that's processed?

Reply:
See https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/runtime/request.go#L153